recipes/configupdater/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "ConfigUpdater" %}
+{% set version = "2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6a60447fb25e5cb5036cdd5761287ac5649135a49094bc8bd71d999417483441
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - configupdater
+  requires:
```

I forgot you'll also need this 😅 
```suggestion
  requires:
    - pip
```